### PR TITLE
RC1

### PR DESCRIPTION
**Pull request by moonglum**
* Update to latest version of faucet-pipeline
* Allow users to import from node_modules via absolute paths
* Allow users to select their browserslist

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ module.exports = (pluginConfig, assetManager, { watcher, browsers, compact }) =>
 function buildConfig(pluginConfig, assetManager, { watcher, browsers, compact }) {
 	return pluginConfig.map(bundleConfig => {
 		let inputFileName = assetManager.resolvePath(bundleConfig.source, {
-			enforceRelative: true
+			enforceRelative: false
```
Comment (FND) [inline lib/index.js]: Might as well omit this `options`(!) object entirely?

(ditto for faucet-static btw)

File: lib/index.js [hunk 2]
```diff
@@ -24,10 +24,21 @@ function buildConfig(pluginConfig, assetManager, { watcher, browsers, compact })
 		let outputStyle = compact ? "compact" : "nested";
 		let includePaths = [ assetManager.packagesDir ];
 
+		let selectedBrowsers;
+
```
Comment (FND) [inline lib/index.js]: FWIW, this blank line seems unhelpful.

File: test/test_no_prefix/faucet.config.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+module.exports = {
+	sass: [{
+		source: "./src/index.scss",
+		target: "./dist/bundle.css",
+		browserslist: false
+	}],
+	manifest: {
+		file: "./dist/manifest.json",
+		value: f => `/assets/${f}`
+	},
```
Comment (FND) [inline test/test_no_prefix/faucet.config.js]: Why are you testing the manifest? Seems like an unnecessary maintenance burden for plugins (except perhaps perhaps for the combination of faucet-sass and faucet-static).

**Comment by moonglum** [comment]
Appeased @FND and rebased. One question that was asked:

* Why am I testing the manifest? Do we test the manifest functionality entirely in core now? Then I can remove some tests from the two repos.

**Comment by FND** [comment]
> Why am I testing the manifest? Do we test the manifest functionality entirely in core now? Then I can remove some tests from the two repos.

No, actually we don't. 😭 Seems like I got momentarily confused there. Nevertheless, seems to me that there are too many tests using the manifest; faucet-js intentionally limits this to a couple of tests.

**Comment by moonglum** [comment]
I think you're right. But even if the manifest was tested decently, I would keep one or two tests in the other projects. I see them as integration tests with another lib: If we change the signature of the file writer, I want the tests of the libs to notice that.

**Comment by FND** [comment]
Agreed. 👍

**Pull request merged**